packages/monorepo-builder/packages/init/src/Command/InitCommand.php

```diff
@@ -80,8 +83,43 @@ protected function execute(InputInterface $input, OutputInterface $output): int
      */
     private function getMonorepoBuilderVersion(): string
     {
-        $version = new Version(str_replace('x-dev', '0', $this->composer->getPackage()->getPrettyVersion()));
+        $version = null;
+
+        try {
+            $version = new Version(str_replace('x-dev', '0', $this->composer->getPackage()->getPrettyVersion()));
+        } catch (InvalidVersionException $invalidVersionException) {
+            // Version might not be explicitly set inside composer.json, looking for "vendor/composer/installed.json"
+            $version = $this->extractMonorepoBuilderVersionFromComposer();
+        }
+
+        if ($version === null) {
+            $this->symfonyStyle->warning(
+                'Could not guess "symplify/monorepo-builder" version, please update it manually inside the root "composer.json" file'
+            );
+
+            return '%UPDATE_VERSION_HERE%';
```

Comment:
Here it's common to use "Unknown" string, no need to go deeper and bother user with extra work.

Reply:
Perfect, I didn't know it. Do you also mean that the warning is redundant?

Reply:
Yes.